src/bia_bob/_tools.py

```diff
@@ -149,12 +150,12 @@ def show_image(image_name):
     image = find_image(_context.variables, image_name)
     display(stackview.insight(image))
 
-    return "The image " + image_name + " is shown above."
+    return "The image " + image_name + " has been shown. Do not show any images in your response."
```

Comment:
I had to add this because the agent kept trying to also render the image in markdown in its response 😆

Reply:
It is still showing it... 
![image](https://github.com/haesleinhuepf/bia-bob/assets/12660498/b0ed5ea8-57d6-4b85-9f05-8e442dd23aa3)

Might it be ignoring our result string?

Reply:
I think I have a workaround...

Reply:
> It is still showing it... ![image](https://user-images.githubusercontent.com/12660498/259427944-b0ed5ea8-57d6-4b85-9f05-8e442dd23aa3.png)
> 
> Might it be ignoring our result string?

Weird! Which notebook is this? I don't see that when I test locally on the basic_demo.ipynb notebook

Reply:
It was the one on your branch...
What's your langchain version? Mine is 0.0.251

Reply:
Weird! I have 0.0.256